pages/index.tsx

```diff
@@ -116,6 +117,13 @@ const useStyles = makeStyles(theme => ({
     boxShadow: theme.shadows[5],
     padding: theme.spacing(10, 15, 3),
   },
+  settingsButton: {
```

Comment:
settingsButton not needed?

Reply:
Nope, just removed the style for it, thanks for the heads up